Dockerfile

```diff
@@ -17,6 +17,7 @@ RUN curl -sSL https://github.com/cnrancher/kube-explorer/releases/download/${KUB
     chmod +x /usr/local/bin/kube-explorer
 
 ENV AUTOK3S_CONFIG /root/.autok3s
+ENV DOCKER_HOSTS unix:///var/run/docker.sock
```

Comment:
The path `unix:///var/run/docker.sock` is the default value of `DOCKER_HOSTS`, no need to be specificed

Reply:
Dockerfile: ENV must have two arguments